### ISSUE TITLE
Ridge coefficient difference

### ISSUE THREAD

**Issue by alexhuth** [GabrielKP/enc#94]
Hello all — First, this is nice work! I'm very glad to see it. I was a bit surprised by the enduring difference between the results using our ridge code and the sklearn implementation. The SVD cutoff should not make nearly that much of a difference—it's mainly a wallclock performance optimization rather than for improving model fitting. But digging a bit further, I think the issue is with the regularization coefficients. 

[My ridge code](https://github.com/alexhuth/ridge/tree/master) uses a ridge coefficient alpha as defined [here](https://en.wikipedia.org/wiki/Ridge_regression#Relation_to_singular-value_decomposition_and_Wiener_filter). This is in contrast to the more standard lambda (as in the standard formulation $(X^T X + \lambda I)^{-1} X^T y$), which is equivalent to alpha squared in the linked definition and in my code. [Sklearn, on the other hand](https://scikit-learn.org/stable/modules/linear_model.html#regression), uses "alpha" defined as in the loss function $||Xw-y||_2^2 + \alpha ||w||_2^2$, which is equivalent to $\alpha^2$ in my code.

I would bet that if you square the alphas going into sklearn.Ridge, the performance of the sklearn version will improve substantially.

Apologies for the confusion!

**Comment by KristijanArmeni** [comment]
Thanks for the input @alexhuth, this is great! It hasn't crossed our minds to look at alphas directly. It'd be great to rerun and see if this checks out. We might (I believe I won't be able myself), but even if we don't manage, we're definitely logging this for any future use. The difference in performance would warrant it.

And thanks for sharing the dataset to begin with, such a great resource, it was great fun hacking around with it!
